### PR TITLE
Refresh 12/11/20

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Security Response
+
+If you've found a security issue that you'd like to disclose confidentially please contact Red Hat's Product Security team.
```
Comment (dockerymick) [inline SECURITY.md:3]: "If you found a security issue that you want to disclose confidentially , contact Red Hat's Product Security team. 

For more details, see [Security Contacts and Procedures](https://access.redhat.com/security/team/contact)"

**Review by dockerymick — APPROVED** [review]
I didn't see the PR for the SECURITY.md file. I made suggestions for that file. If you agree with the change, i can update 2.1_stage

File: SECURITY.md (hunk 1 above)
Comment (cadawson) [reply SECURITY.md:3]: Sorry, I didn't see that. I can make that change and rerun this.

**Comment by cadawson** [comment]
Closed this PR to add a change.

**Pull request closed without merging**